### ISSUE TITLE
New false negatives in infix_spaces_linter

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#1342]
The following throws a lint in 2.0.1 but no longer does:

```
lint("x +
y+
z", infix_spaces_linter()
)
```

`y+` should be linted
